feat(ai): enable worker image pulling

rickstaa opened the pull request:
**What does this pull request do? Explain your changes. (required)**
<!-- A clear and concise description of what this pull request does. -->

This commit ensures that the docker containers the worker uses are
pulled during startup. They use the new changes implemented in
https://github.com/livepeer/ai-worker/pull/200.

**Specific updates (required)**
<!--- List out all significant updates your code introduces -->
- Calls the new `EnsureImageAvailable` implemented in https://github.com/livepeer/ai-worker/pull/200 during orchestrator/worker startup. This will ensure that the worker has all the images it needs to serve the specifie requests.

**How did you test each of these updates (required)**
<!-- A detailed description of how you tested your code changes. Include details of your testing environment, and the tests you ran to see how your change affects other areas of the code, etc. -->

- I ran offchain e2e tests to see if the requests were working.

**Does this pull request close any open issues?**
<!-- Fixes # -->

**Checklist:**
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] Read the [contribution guide](./CONTRIBUTING.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [ ] [Pending changelog](./CHANGELOG_PENDING.md) updated

rickstaa committed:
feat(ai): enable worker image pulling

This commit ensures that the docker containers the worker uses are
pulled during startup. They use the new changes implemented in
https://github.com/livepeer/ai-worker/pull/200.

rickstaa commented:
@leszko we have to update the worker version before merging. This is why it is still a draf. Apart from that we are good to go.

leszko commented on cmd/livepeer/starter/starter.go:
```diff
@@ -1236,6 +1236,9 @@ func StartLivepeer(ctx context.Context, cfg LivepeerConfig) {
 				}
 
+				// Ensure the AI worker has the image needed to serve the job.
+				n.AIWorker.EnsureImageAvailable(ctx, config.Pipeline, config.ModelID)
```
Could you handle error here?

leszko reviewed (approved):
Added one comment, other than that LGTM

rickstaa committed:
refactor: handle image pull error

This commit ensures that we corectly handle errors that are thrown when
pulling the required pipeline images.

rickstaa committed:
test(ai): fix tests

This commit ensures that everything is mocked correctly for the tests to
pass.

rickstaa committed:
chore(ai): update CHANGELOG_PENDING
